main.go

```diff
@@ -178,6 +187,7 @@ var manCmd = &cobra.Command{
 func init() {
 	rootCmd.Flags().StringVarP(&name, "name", "n", "", "timer name")
 	rootCmd.Flags().BoolVarP(&altscreen, "fullscreen", "f", false, "fullscreen")
+	rootCmd.Flags().BoolVarP(&twentyFourHourStartTime, "24", "", false, "Use 24-hour format for start time")
```

Comment:
maybe it would be nicer to do something like:

```
--format 24h
```

```
--format kitchen
```

etc

that way we could add more formats in the future... or even allow the user to provide their own.

wdyt?

Reply:
Yes, I thought about it, but was a bit reluctant to leak implementation-specific details such as "kitchen" meaning that specific format. But if you prefer it, I could modify the PR...

Reply:
yes plz

Reply:
Done, I'll squash the fixup commits before potential merge.